dockerfiles/base-ci/Dockerfile

```diff
@@ -62,7 +62,10 @@ RUN set -eux; \
 # apt clean up
 	apt-get autoremove -y && \
 	apt-get clean && \
-	rm -rf /var/lib/apt/lists/*
+  rm -rf /var/lib/apt/lists/* && \
+# add non-root user
+  groupadd -g 1000 nonroot && \
+  useradd -u 1000 -g 1000 -s /bin/bash -m nonroot
```

Comment:
I've read somewhere that it's generally safer to use ID > 10k to not to accidentally collapse with something, does it worth it?

Another source of "best practices" was even claiming that IDs are obsolete, are you playing safe here or is it better this way?

Reply:
This is valid because we output caches and artifacts that are reused by the other containers and CI processes.